src/ADAL.PCL/HttpClientWrapper.cs

```diff
@@ -158,7 +158,7 @@ public int TimeoutInMilliSeconds
                 }
             }
 
-            return new HttpWebResponseWrapper(await response.Content.ReadAsStreamAsync().ConfigureAwait(false), headers, response.StatusCode);
+            return new HttpWebResponseWrapper(await response.Content.ReadAsStringAsync(), headers, response.StatusCode);
```

Comment:
missing ConfigureAwait(false). We must be careful about not removing configureawait from the code.